model/host/hostutil.go

```diff
@@ -341,24 +338,36 @@ func (h *Host) ForceReinstallJasperCommand(settings *evergreen.Settings) string
 		params = append(params, fmt.Sprintf("--precondition=%s", ps.Path))
 	}
 
-	return h.jasperServiceCommand(settings.HostJasper, jcli.ServiceForceReinstallCommand, params...)
+	cmdEnv := map[string]string{}
+	if h.Distro.BootstrapSettings.ServiceUser != "" && h.ServicePassword != "" {
+		cmdEnv["JASPER_USER_PASSWORD"] = h.ServicePassword
```

Comment:
nit: prefer to make this a constant string in `globals.go`

Reply:
It's not used anywhere else except this one spot, so I made it a constant in `hostutil.go`.